impl/src/main/java/org/glassfish/soteria/mechanisms/HttpMessageContextImpl.java

```diff
@@ -16,29 +16,29 @@
 
 package org.glassfish.soteria.mechanisms;
 
-import static javax.security.enterprise.AuthenticationStatus.NOT_DONE;
-import static javax.security.enterprise.AuthenticationStatus.SEND_CONTINUE;
-import static javax.security.enterprise.AuthenticationStatus.SEND_FAILURE;
-import static javax.security.enterprise.AuthenticationStatus.SUCCESS;
-import static javax.security.enterprise.identitystore.CredentialValidationResult.Status.VALID;
-import static javax.servlet.http.HttpServletResponse.SC_NOT_FOUND;
-import static javax.servlet.http.HttpServletResponse.SC_UNAUTHORIZED;
+import static jakarta.security.enterprise.AuthenticationStatus.NOT_DONE;
+import static jakarta.security.enterprise.AuthenticationStatus.SEND_CONTINUE;
+import static jakarta.security.enterprise.AuthenticationStatus.SEND_FAILURE;
+import static jakarta.security.enterprise.AuthenticationStatus.SUCCESS;
+import static jakarta.security.enterprise.identitystore.CredentialValidationResult.Status.VALID;
+import static jakarta.servlet.http.HttpServletResponse.SC_NOT_FOUND;
+import static jakarta.servlet.http.HttpServletResponse.SC_UNAUTHORIZED;
 
 import java.io.IOException;
 import java.security.Principal;
 import java.util.Set;
 
 import javax.security.auth.Subject;
```

Comment:
Non javadoc comments still refer to javax package name of security API classes.

Reply:
Thanks @rdebusscher 

In fact those comments were wrong to begin with, as they were still pointing to a very old version of EE Security. I removed them, as they were 1. Wrong, 2. Not needed (override already merges in the javadoc from the interface)